main.go

```diff
@@ -19,23 +20,42 @@ var (
 	GitCommit string
 	// Version represents the version of the service that is running
 	Version string
+
+/*
```

Comment:
Is this comment an example of how the vars look like? Or shall it be removed otherwise?

Reply:
Its an example of values to set the variables to so that you can debug the application in vscode.
(did you read my commit comments ?)

Reply:
Yes, sorry, I didn't read the last point